README.md

```diff
@@ -113,7 +146,11 @@ type ListenHandler<T> = (PushCallback<T>, StopCallback<T>, FailCallback<T>) => v
 type RemoveHandler<T> = (PushCallback<T>, StopCallback<T>, FailCallback<T>) => void
 
 /* High water mark defaults to 100. Set to undefined to disable warnings. */
-interface EventIteratorOptions = {highWatermark?: number }
+interface EventIteratorOptions = {
+  highWatermark?: number,
+  onPause?: Function,
```

Comment:
Suggested alternative names: `onDrain` and `onFill`

Reply:
During refactoring to these names I got myself confused with which is supposed to be onDrain and which is supposed to be onFill. I think I actually prefer the `onPause` and `onResume` callbacks.